### PR TITLE
Adds documentation to devices data transmission rate

### PR DESCRIPTION
**Pull request by waldemarnt**
The idea here was to document how frequently the data is collected by the Devices on the machinery.
And explain the differences between data plans like Standard Plus and Advanced.
Updated:
screen shot 2016-08-10 at 7 12 36 pm

### CODE REVIEW

**Comment by lucaslago** [comment]
Would every person reading the documentation know what `devices` mean? Would it be of value explaining what is a device?

File: source/includes/telemetry_api/_devices.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+The [Telemetry API](#telemetry-api) is supplied by `Topcon` and `Gateway` devices.
+
+`AGCO` service's offers [data plans](#data-plans) that are related to how often the data is collected by the `Device` on the machine
+and how often this data is sent to the `Telemetry API`.
+
+### Data harvesting
+
+The data harvesting is separated in two steps:
+
+1. The first step the `CAN variables data` are collected through the machine in an interval of time.
```
Comment (lucaslago) [inline source/includes/telemetry_api/_devices.md]: s/The/On the first

Comment (lucaslago) [inline source/includes/telemetry_api/_devices.md]: *s/The/In the/

File: source/includes/telemetry_api/_devices.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+This inverval of time depends of which plan the `Equipment` have. This step is called `Data Logging Rate`.
```
Comment (lucaslago) [inline source/includes/telemetry_api/_devices.md]: s/of/on

File: source/includes/telemetry_api/_devices.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+2. The second step is the syncronization with the server. The often of this syncronization also
```
Comment (lucaslago) [inline source/includes/telemetry_api/_devices.md]: s/syncronization/synchronization

Comment (lucaslago) [inline source/includes/telemetry_api/_devices.md]: s/The often/How often this synchronization occurs also depends on the Equipment's data plan (Standard Plus or Advanced).

**Comment by waldemarnt** [comment]
We need have to include the vendor names to proceed here.

**Comment by dclucas** [comment]
Suggestion: why don't we publish this as-is, while we get the verbiage sorted out? The current-state doc is already pretty good...

File: source/includes/telemetry_api/_devices.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+`AGCO` service's offers [data plans](#data-plans) that are related to how often the data is collected by the `Device` on the machine
+and how often this data is sent to the `Telemetry API`.
+
+### Data harvesting
+
+The data harvesting is separated in two steps:
+
+1. In the first step the `CAN variables data` are collected through the machine in an interval of time.
+This inverval of time depends on which plan the `Equipment` have. This step is called `Data Logging Rate`.
```
Comment (jeffhsta) [inline source/includes/telemetry_api/_devices.md]: s/This inverval of time depends on which plan the `Equipment` have./This inverval of time depends on which plan is available for the needed `Equipment`.

Comment (waldemarnt) [inline source/includes/telemetry_api/_devices.md]: I changed to This inverval of time depends on which the Equipment belongs to. It's ok?

**Comment by lucaslago** [comment]
👍

**Comment by jeffhsta** [comment]
:+1:

**Comment by carolinaborim** [comment]
👍

**Pull request merged**